.vscode/settings.json

```diff
@@ -12,6 +12,10 @@
         "*.c": "c",
         "*.cpp": "cpp",
         "*.hpp": "cpp",
-        "xstddef": "c"
+        "xstddef": "c",
```

Comment:
Can you pull these out? If these are good changes we can accept them in another PR, but we don't like to group unrelated changes in.